src/test/scala/nl.knaw.dans.easy.deposit/authentication/SessionSpec.scala

```diff
@@ -185,11 +188,23 @@ class SessionSpec extends TestSupportFixture with ServletFixture with ScalatraSu
     ) {
       body shouldBe ""
       header("REMOTE_USER") shouldBe ""
-      header("Set-Cookie") should startWith("scentry.auth.default.user=;")
       status shouldBe NO_CONTENT_204
+      // TODO basic authentication is intended for internal test purposes
+      //  if exposed beyond the firewall we need to prevent the first cookie
+      shouldHaveEmptyCookie(2)
```

Comment:
challenge to get just the set-cookie that cleans up: enable the following line only for the DepositServlet
        https://github.com/janvanmansum/easy-deposit-api/blob/15f8697c40241253e12f1082eb8e850b2ed1361d/src/main/scala/nl.knaw.dans.easy.deposit/authentication/AuthenticationSupport.scala#L83